network/router.go

```diff
@@ -231,6 +256,19 @@ func (r *Router) handleConn(remote *ServerIdentity, c Conn) {
 	for {
 		packet, err := c.Receive()
 
+		// Be careful not to hold r's mutex while
+		// pausing, or else Unpause would deadlock.
+		r.Lock()
+		paused := r.paused
+		r.Unlock()
+		if paused != nil {
+			<-paused
```

Comment:
How does that behave if a new node tries to connect?
Is this a good simulation of a node being down? Or do we already get some of the TCP handshake happening and thus not simulate a node being down?

Reply:
I agree it is not ideal, but it serves the current need. I propose using this until we need something better, then doing that and replacing this.